src/sidebar/services/annotations.ts

```diff
@@ -232,11 +250,17 @@ export class AnnotationsService {
    * the annotation's `Draft` will be removed and the annotation added
    * to the store.
    */
-  async save(annotation: Annotation) {
+  async save(
+    annotation: Annotation,
+    mentionsOptions: MentionsOptions = { mentionMode: 'username' },
```

Comment:
This param should be mandatory, as some cases where annotations are saved are currently not taking into consideration the mention mode:

1. Autosave.
2. Annotations import.

I'll address those separately.